### PR TITLE
Liepa popescu/feature/t2 verify all user roles log

### PR DESCRIPTION
**Pull request by Liepapa**

### CODE REVIEW

File: cypress/integration/login.js [hunk 1]
```diff
@@ -1,27 +1,134 @@
...
+ 
+        cy.url().should('include', '/time-logging');
+        cy.get('.page__title').contains('Timesheets')
+        cy.get('.calendar').should('be.visible')
+        cy.get('.tile.form').should('be.visible')
+        cy.get('.user-info__title').contains('Liepa Popescu');
+        cy.get('.main-nav').find('li').should('have.length', 5);
+        cy.get('.main-nav__link--active').should('have.css','color', 'rgb(64, 76, 237)');
+ 
+    })
```
Comment (DavidV44) [inline cypress/integration/login.js]: Different "it" block should be separated with an empty line

File: cypress/integration/login.js [hunk 2]
```diff
@@ -1,27 +1,134 @@
...
+ 
+        cy.url().should('include', '/time-logging');
+        cy.get('.page__title').contains('Timesheets')
+        cy.get('.calendar').should('be.visible')
+        cy.get('.tile.form').should('be.visible')
+        cy.get('.user-info__title').contains('Liepa Popescu');
+        cy.get('.main-nav').find('li').should('have.length', 2);
+        cy.get('.main-nav__link--active').should('have.css','color', 'rgb(64, 76, 237)');
+ 
+    })
```
Comment (DavidV44) [inline cypress/integration/login.js]: Different "it" block should be separated with an empty line

**Review by DavidV44 — CHANGES_REQUESTED** [review]

File: cypress/integration/login.js [hunk 3]
```diff
@@ -1,27 +1,134 @@
...
         cy.url().should('include', '/time-logging');
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
+        cy.get('.user-info__title').contains('Liepa Popescu');
         cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.main-nav__link--active').should('have.css','color', 'rgb(64, 76, 237)');
+ 
+    })
```
Comment (DavidV44) [inline cypress/integration/login.js]: Different "it" block should be separated with an empty line

**Review by DavidV44 — CHANGES_REQUESTED** [review]

File: cypress/integration/login.js [hunk 4]
```diff
@@ -1,27 +1,134 @@
...
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
+        cy.get('.user-info__title').contains('Liepa Popescu');
         cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.main-nav__link--active').should('have.css','color', 'rgb(64, 76, 237)');
+    })
+
+    it('Should display validation for empty user after attempted loggin', function () {
```
Comment (DavidV44) [inline cypress/integration/login.js]: This same test is copied multiple times, i think one is enough :)

**Review by DavidV44 — CHANGES_REQUESTED** [review]

**Review by nikolaj-tolkaciov — APPROVED** [review]

**Pull request merged**